procedures/auxtel/reduction_procedures/PSF_evaluation.ipynb

```diff
@@ -71,7 +71,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dataId = {'day_obs': 20220505, 'seq_num': 902, 'detector': 0, \"instrument\": 'LATISS'}"
+    "dataId = {'day_obs': 20220914, 'seq_num': 750, 'detector': 0, \"instrument\": 'LATISS'}"
```

Comment:
This should be a frame that is in the test dataset... just so it always runs.
I believe the following should work:
` "dataId = {'day_obs': 20200315, 'seq_num': 120, 'detector': 0, \"instrument\": 'LATISS'}" `

Reply:
Done